util/test/paratest.client

```diff
@@ -10,7 +10,7 @@
 #  testdir - directory to run start_test on
 #  chplenv - Chapel environment variables
 #  futures-mode - include .futures (0=no, 1=all, 2=futures only, 3=futures with skipifs)
-#  valgrind - run valgrind (0=no, 1=yes)
+#  valgrind - run valgrind (0=no, 1=yes, 2=execs only)
```

Comment:
I would change the "yes" here to "comp and exec" or something similar for clarity.